datasets/us/dos_terrorist_exclusion/us_dos_terrorist_exclusion.yml

```diff
@@ -0,0 +1,206 @@
+        names:
+          - International Battalion
+          - Islamic Peacekeeping International Brigade
+          - Peacekeeping Battalion
+          - The International Brigade
+          - The Islamic Peacekeeping Army
+          - The Islamic Peacekeeping Brigade
+      - match: The Lord’s Resistance Army (LRA)
+        names:
+          - LRA
+      - match: The Riyadus-Salikhin Reconnaissance and Sabotage Battalion of Chechen Martyrs (a.k.a. Riyadus-Salikhin Reconnaissance and Sabotage Battalion, a.k.a. Riyadh-as-Saliheen, a.k.a. the Sabotage and Military Surveillance Group of the Riyadh al-Salihin Martyrs, a.k.a. Riyadus-Salikhin Reconnaissance and Sabotage Battalion of Shahids (Martyrs))
+        names:
+          - Riyadus-Salikhin Reconnaissance and Sabotage Battalion
+          - Riyadh-as-Saliheen
+          - the Sabotage and Military Surveillance Group of the Riyadh al-Salihin Martyrs
+          - Riyadus-Salikhin Reconnaissance and Sabotage Battalion of Shahids
+      - match: The Special Purpose Islamic Regiment (a.k.a. the Islamic Special Purpose Regiment, a.k.a. the al-Jihad-Fisi-Sabililah Special Islamic Regiment, a.k.a. Islamic Regiment of Special Meaning)
+        names:
+          - the Islamic Special Purpose Regiment
+          - the al-Jihad-Fisi-Sabililah Special Islamic Regiment
+          - Islamic Regiment of Special Meaning
+      - match: Tunisian Combat Group (a.k.a. GCT, a.k.a. Groupe Combattant Tunisien, a.k.a. Jama’a Combattante Tunisien, a.k.a. JCT; a.k.a. Tunisian Combatant Group)
+        names:
+          - GCT
+          - Groupe Combattant Tunisien
+          - Jama’a Combattante Tunisien
+          - JCT
+          - Tunisian Combatant Group
+      - match: Ulster Defense Association (a.k.a. Ulster Freedom Fighters)
+        names:
+          - Ulster Freedom Fighters
+      - match: Ummah Tameer E-Nau (UTN) (a.k.a. Foundation for Construction; a.k.a. Nation Building; a.k.a. Reconstruction Foundation; a.k.a. Reconstruction of the Islamic Community; a.k.a. Reconstruction of the Muslim Ummah; a.k.a. Ummah Tameer I-Nau; a.k.a. Ummah Tameer E-Nau; a.k.a. Ummah Tameer-I-Pau)
+        names:
+          - Foundation for Construction
+          - Nation Building
+          - Reconstruction Foundation
+          - Reconstruction of the Islamic Community
+          - Reconstruction of the Muslim Ummah
+          - Ummah Tameer I-Nau
+          - Ummah Tameer E-Nau
+          - Ummah Tameer-I-Pau
+      - match: First of October Antifascist Resistance Group (GRAPO) (a.k.a. Grupo de Resistencia Anti-Fascista Premero De Octubre)
+        names:
+          - GRAPO
+          - Grupo de Resistencia Anti-Fascista Premero De Octubre
+assertions:
+  min:
+    schema_entities:
+      LegalEntity: 56
+  max:
+    schema_entities:
+      LegalEntity: 56
```

Comment:
make these a little bit looser so we don't have to change this immediately when they add/remove someone.

> A good rule of thumb for datasets that change over time is minima 10% below the expected number to allow normal variation, unless there's a known hard minimum, and a maximum around 20% above the expectation to leave room to grow, if the number is expected to fluctuate.

https://zavod.opensanctions.org/metadata/#data-assertions